docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contribution Guidelines for Aether
+
+## Welcome!
+
+We are happy to have contributors to our efforts to make IT sustainable. To get
+started please check out any documentation including our [methodologies][1]
+
+## Code Of Conduct
+
+We adhere to the [contributors covenant][4]
+
+the terms of this covenant will be subject to appropriate consequences, including but not limited to warnings, temporary bans, or permanent expulsion from the project community.
+
+Our community values inclusivity, respect, and collaboration. We strive to create a welcoming environment for everyone, regardless of race, ethnicity, gender identity, sexual orientation, religion, disability, age, nationality, or any other personal characteristic.
+
+We encourage constructive criticism, feedback, and discussions aimed at improving the project. However, we do not tolerate harassment, discrimination, personal attacks, or any form of disrespectful behavior.
+
+We ask all contributors to:
+
+1. Treat others with kindness, empathy, and respect.
+2. Listen actively and considerately to diverse perspectives.
+3. Avoid offensive language, derogatory remarks, or inappropriate imagery.
+4. Respect differing opinions and gracefully resolve disagreements.
+5. Use clear and inclusive language in all communications.
+6. Refrain from engaging in any form of harassment, including but not limited to verbal, physical, sexual, or online harassment.
+7. Be mindful of the impact of your words and actions on others.
+8. Prioritize the well-being and safety of all community members.
+
+By participating in this project, you agree to abide by these guidelines and uphold the principles of our community. Together, we can create a positive and inclusive environment where everyone feels valued and empowered to contribute.
+
+
+## Local Environment
```

Comment:
I'm not sure this should be here and the README, when things change they will need to change in multiple places. Maybe we should have a separate doc from both that is for local environment and setup